### PR TITLE
[plugin.video.rocketbeans] 1.0.0

### PR DESCRIPTION
**Pull request by Razzeee**
### Description

Basically wrapping the youtube plugin and adding some shedule reader via an api.

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

**Comment by Razzeee** [comment]
@enen92 in that case we need to update the readme https://github.com/xbmc/repo-plugins/blob/jarvis/README.md#status

**Comment by enen92** [comment]
I'm guiding myself by the wiki: http://kodi.wiki/view/Submitting_Add-ons

**Comment by Razzeee** [comment]
thought that would be the case, well one of them is not matching then
@MartijnKaijser what's correct?

**Comment by enen92** [comment]
pinging @ronie and @phil65 as well

**Comment by ronie** [comment]
wiki is always correct.

i wouldn't add this info to the readme, unless someone enjoys updating them for all repos with every release.

**Comment by Razzeee** [comment]
Switched to krypton as a base

File: plugin.video.rocketbeans/addon.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.rocketbeans" version="1.0.0" name="Rocket Beans TV" provider-name="Razzeee">
+    <requires>
+        <import addon="xbmc.python" version="2.14.0" />
```
Comment (enen92) [inline plugin.video.rocketbeans/addon.xml]: bump

File: plugin.video.rocketbeans/addon.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        <import addon="plugin.video.youtube" version="5.2.1" />
+        <import addon="script.module.routing" version="0.2.0"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="addon.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="de_de">Rocket Beans TV | 24/7 Entertainment</summary>
```
Comment (enen92) [inline plugin.video.rocketbeans/addon.xml]: I think you are required to provide the same tags in en_GB as well (for the website and addon browser)

File: plugin.video.rocketbeans/addon.xml (hunk 1 above)
Comment (Razzeee) [inline plugin.video.rocketbeans/addon.xml]: Yeah, that sliped between the cracks now

**Comment by Razzeee** [comment]
Updated

**Pull request merged**